lib/messaging/messagehandler.rb

```diff
@@ -22,7 +28,7 @@ class MessageHandler
 
   def initialize
     #@dbaccess = DbAccess.new
-    @dbaccess = $dbaccess
+    @dbaccess = $dbaccess || NullDatabase.new
```

Comment:
Yes. I'll push a file for this soon.

On Thu, Feb 26, 2015 at 1:44 PM, Rick Carlino <notifications@github.com>
wrote:

> In lib/messaging/messagehandler.rb
> <https://github.com/FarmBot/farmbot-raspberry-pi-controller/pull/54#discussion_r25423916>
> :
>
> > @@ -22,7 +28,7 @@ class MessageHandler
> >
> >    def initialize
> >      #@dbaccess = DbAccess.new
> > -    @dbaccess = $dbaccess
> > +    @dbaccess = $dbaccess || NullDatabase.new
>
> So if I moved this into its own process, I could remove the reference to
> the database completely?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FarmBot/farmbot-raspberry-pi-controller/pull/54/files#r25423916>
> .
>